src/chat/chat_params.rs

```diff
@@ -0,0 +1,289 @@
+use makepad_widgets::*;
+
+use crate::data::store::Store;
+
+live_design! {
+    import makepad_widgets::base::*;
+    import makepad_widgets::theme_desktop_dark::*;
+
+    import crate::shared::styles::*;
+    import crate::shared::widgets::FadeView;
+    import crate::shared::widgets::MoxinButton;
+    import crate::shared::widgets::MoxinSlider;
+    import crate::shared::widgets::MoxinSwitch;
+    import crate::shared::widgets::MoxinTextInput;
+    import makepad_draw::shader::std::*;
+
+
+    // TODO: Use proper icons for right panel.
+    ICON_CLOSE_PANEL = dep("crate://self/resources/icons/open_left_panel.svg")
+    ICON_OPEN_PANEL = dep("crate://self/resources/icons/close_left_panel.svg")
```

Comment:
I tried copying as svg [this](https://www.figma.com/design/LIaqW5HF3CGtq4XMpDxDMD/MoxinUI---LLM-Desktop-App?node-id=1885-2756&t=WFGkMXe5bBlBHE3I-4) but it blows as shown in the following video, so I'm using the same icons that are used on the left side.

https://github.com/moxin-org/moxin/assets/7684329/ce14d91d-c37c-4406-8d53-e9d94cca45f5